src/main/kotlin/org/rust/cargo/commands/Cargo.kt

```diff
@@ -53,6 +55,7 @@ class Cargo(
             .withParameters(command)
             .withParameters(additionalArguments)
             .withEnvironment(environmentVariables)
+            .withEnvironment(CargoConstants.RUSTC_ENV_VAR, pathToRustExecutable)
```

Comment:
Does not matter much, this is one commit length pull request, so "squash merge" would work anyway.